README.md

```diff
@@ -1,2 +1,36 @@
-# chaostoolkit-extension-template
-Template starting point for a new Python-based Chaos Toolkit extension
+# Chaos Toolkit Extension Template
```

Comment:
should the README be targetting those who want to apply the template or the users of the resulting project?

Reply:
For now I'll leave as-is as I think this README content will migrate into custom-extension authoring documentation on chaostoolkit.org